security/src/main/java/com/networknt/security/JwtVerifier.java

```diff
@@ -202,7 +211,7 @@ public static String getJwtFromAuthorization(String authorization) {
      * @throws ExpiredTokenException throw when the token is expired
      */
     public JwtClaims verifyJwt(String jwt, boolean ignoreExpiry, boolean isToken) throws InvalidJwtException, ExpiredTokenException {
-        return verifyJwt(jwt, ignoreExpiry, isToken, this::getKeyResolver);
+        return verifyJwt(jwt, isToken, this::getKeyResolver);
```

Comment:
@mihai-vladuc 
Will this affect backwards compatability? You have changed the API of JwtVerifier.verifyJwt... You also made an API change on line 233